src/init.cpp

```diff
@@ -421,11 +421,18 @@ bool AppInit2(ThreadHandlerPtr threads)
     }
 
     fDebug3 = false;
+
     if (GetArg("-debug3", "false")=="true")
     {
             fDebug3 = true;
             LogPrintf("Entering GRC debug mode 3.\n");
     }
+
+    fDebug4 = GetBoolArg("-debug4");
```

Comment:
reminder to self to join this to same as other debugs or change others to appropriate method